packages/ra-ui-materialui/src/form/TabbedFormView.tsx

```diff
@@ -36,7 +36,7 @@ export const TabbedFormView = (props: TabbedFormViewProps): ReactElement => {
         ...rest
     } = props;
     const location = useLocation();
-    const resolvedPath = useResolvedPath('');
+    const resolvedPath = useResolvedPath('.');
```

Comment:
Unfortunately this part doesn't work:
- React-router 6.18 requires `useResolvedPath('.')`
- While React-router 6.20 requires `useResolvedPath('..')`

I can't find a way to make both versions happy with the same code...

I'm open to any suggestions if any, or, you may want to upvote https://github.com/remix-run/react-router/issues/11048#issuecomment-1829349584 and https://github.com/remix-run/react-router/issues/11052#issuecomment-1829467624 to request for a fix from react-router.

Reply:
I suggest we change the package.json to forbid recent react-router versions.